### PR TITLE
adding Photos++ 3.55

### PR DESCRIPTION
**Pull request by inugent**
This pull request replaces PR 568. It is going to CMSSW_7_2_X/stable to allow for testing of the new PhotosppInterface. We expect to replace this version once there is a stable version of Photos++.  The libraries were tested to compile with cmsdist CMSSW_7_2_X/stable.

### CODE REVIEW

**Comment by ghost** [comment]
Verified it compiles correctly.
The addition should be discussed during next ORP meeting.

**Comment by vciulli** [comment]
Sorry. We forgot to discuss it in the meeting. Can this request be merged?

**Comment by ktf** [comment]
@degano, did you test up to cmssw-tool-conf?

**Comment by ghost** [comment]
I did now, everything seems to work fine.

**Comment by vciulli** [comment]
Would it be possible to merge this pull in the IB? 
There are other pulls waiting for this one to be tested
Thanks

**Comment by bendavid** [comment]
bump

**Comment by inugent** [comment]
Hi Josh,
  Can you fill me in on the new plan for this pull? Thanks,
Ian

**Comment by bendavid** [comment]
The plan is for it to be merged into 72x.

File: photospp.spec [hunk 1]
```diff
@@ -0,0 +1,42 @@
+### RPM external photos 3.55
```
Comment (ktf) [inline photospp.spec:1]: this is really photospp, right?

**Comment by ktf** [comment]
I'm trying this and others out right now.

**Comment by ktf** [comment]
Fixed in #820. Closing.

**Pull request closed without merging**